scss/_card.scss

```diff
@@ -200,10 +200,12 @@
 
           .card-img-top,
           .card-header {
+            // stylelint-disable-next-line property-blacklist
             border-top-right-radius: 0;
```

Comment:
Not sure if we should add a mixin for `border-top-right-radius` and the other specific border radii?

Reply:
Eh, probably not necessary? It'd save on a bit of CSS when folks compile without rounded corners enabled, but probably not worth it right now.